baremetal/metal3machine_manager_test.go

```diff
@@ -3001,6 +3001,14 @@ var _ = Describe("Metal3Machine manager", func() {
 			},
 			expectClaim: true,
 		}),
+		Entry("DataClaim is empty", testCaseM3MetaData{
+			M3Machine: newMetal3Machine("myName", nil, &capm3.Metal3MachineSpec{
+				DataTemplate: &corev1.ObjectReference{Name: "abcd"},
+			}, nil, nil),
+			Machine:     newMachine("myName", "myName", nil),
```

Comment:
Would it be clearer if you change the myName to resource oriented naming. 
Eg. 
myNamespace1, myMachine1.....

Reply:
+1 with @Xenwar's suggestion

Reply:
I will look into this, we have seen these practice many times in different test entries.

Reply:
"myName" string has been used in many other entries more that 80 times. @Xenwar . So I think I need to keep this for regularity otherwise need to change in all other entries.